### PR TITLE
Enables alternative correlation methods

### PR DESCRIPTION
**Pull request by cristineguadelupe**
I'm passing the correlation method as string to follow the `Series.rank` convention but I'd suggest to use atoms instead. WDYT?

### CODE REVIEW

File: lib/explorer/data_frame.ex [hunk 1]
```diff
@@ -5678,6 +5678,9 @@ defmodule Explorer.DataFrame do
   * `:column_name` - the name of the column with column names. Defaults to "names".
   * `:ddof` - the 'delta degrees of freedom' - the divisor used in the correlation
     calculation. Defaults to 1.
+  * `method` refers to the correlation method. The following methods are available:
```
Comment (josevalim) [inline lib/explorer/data_frame.ex:5681]:
```suggestion
  * `:method` refers to the correlation method. The following methods are available:
```

**Comment by josevalim** [comment]
Looks great but indeed atoms will be better. I think you need to declare an enum in Rust. Ask @philss, he will know for sure!

**Review by philss — APPROVED** [review]
:shipit:

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**